extensions/sql-migration/src/wizard/databaseBackupPage.ts

```diff
@@ -408,14 +409,55 @@ export class DatabaseBackupPage extends MigrationWizardPage {
 	public async onPageEnter(): Promise<void> {
 		await this.getSubscriptionValues();
 		this.wizard.registerNavigationValidator((pageChangeInfo: azdata.window.WizardPageChangeInfo) => {
+			if (pageChangeInfo.newPage < pageChangeInfo.lastPage) {
+				this.wizard.message = { text: '' };
+				return true;
+			}
+			const errors: string[] = [];
+			switch (this.migrationStateModel.databaseBackup.networkContainerType) {
+				case NetworkContainerType.NETWORK_SHARE:
+					if ((<azdata.CategoryValue>this._networkShareContainerSubscriptionDropdown.value).displayName === constants.NO_SUBSCRIPTIONS_FOUND) {
+						errors.push(constants.INVALID_SUBSCRIPTION_ERROR);
+					}
+					if ((<azdata.CategoryValue>this._networkShareContainerStorageAccountDropdown.value).displayName === constants.NO_STORAGE_ACCOUNT_FOUND) {
+						errors.push(constants.INVALID_STORAGE_ACCOUNT_ERROR);
+					}
+					break;
+				case NetworkContainerType.BLOB_CONTAINER:
+					if ((<azdata.CategoryValue>this._blobContainerSubscriptionDropdown.value).displayName === constants.NO_SUBSCRIPTIONS_FOUND) {
+						errors.push(constants.INVALID_SUBSCRIPTION_ERROR);
+					}
+					if ((<azdata.CategoryValue>this._blobContainerStorageAccountDropdown.value).displayName === constants.NO_STORAGE_ACCOUNT_FOUND) {
```

Comment:
I'd suggest making a helper method for getting the error since the logic for picking the error message to display is always the same